### DISCUSSION TITLE
[2021.08.31] 제로

### DISCUSSION THREAD

**Discussion by leechang4** [pyunji/Hyundai-ITnE-Team_QuadCore#87] [Algorithm]
[제로](https://www.acmicpc.net/problem/10773)

**Comment by pyunji** [comment]
[제로 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/BOJ/BOJ_10773.java)

**Comment by hyk339** [comment]
[제로 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/baekjoon/BOJ_10773.java)

**Comment by leechang4** [comment]
[제로 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/BOJ/BOJ%2010773.java)

**Comment by chaeyeon4u** [comment]
[제로 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/BOJ_10773.java)
